src/components/Photo/DocumentLiveCapture.js

```diff
@@ -0,0 +1,164 @@
+// @flow
+import * as React from 'react'
+import { h, Component } from 'preact'
+import { screenshot } from '~utils/camera.js'
+import { mimeType } from '~utils/blob.js'
+import { sendEvent } from '../../Tracker'
+import { DocumentOverlay } from '../Overlay'
+import { ToggleFullScreen } from '../FullScreen'
+import Timeout from '../Timeout'
+import Camera from '../Camera'
+import CameraError from '../CameraError'
+import style from './style.css'
+
+type State = {
+  hasBecomeInactive: boolean,
+  hasCameraError: boolean,
+  snapshotBuffer: Array<{
+    blob: Blob
+  }>,
+}
+
+type Props = {
+  translate: (string, ?{}) => string,
+  onCapture: Function,
+  renderFallback: Function,
+  isUploadFallbackDisabled: boolean,
+  trackScreen: Function,
+  documentType: string,
+  className: string,
+  containerClassName: string,
+  renderTitle: Function,
+  renderError: Function
+}
+
+const screenshotQuality: number = 0.95
+
+export default class DocumentLiveCapture extends Component<Props, State> {
+  webcam = null
+  snapshotIntervalRef: ?IntervalID = null
+
+  state: State = {
+    hasBecomeInactive: false,
+    hasCameraError: false,
+    snapshotBuffer: [],
+  }
+
+  handleTimeout = () => this.setState({ hasBecomeInactive: true })
+
+  handleCameraError = () => this.setState({ hasCameraError: true })
+
+  inactiveError = () => ({
+    name: this.props.isUploadFallbackDisabled ? 'CAMERA_INACTIVE_NO_FALLBACK' : 'CAMERA_INACTIVE',
+    type: 'warning'
+  })
+
+  captureDocument = (blob: Blob, sdkMetadata: Object) => {
+    const document = {
+      blob,
+      sdkMetadata,
+      filename: `document_capture.${mimeType(blob)}`
+    }
+    /* Attempt to get the 'ready' snapshot. But, if that fails, try to get the fresh snapshot - it's better
```

Comment:
Are we going to use multiple frame capture for document too?

Reply:
I think by user tracking we usually mean tracking analytics in Woopra. Anyways, I think multiple frame capture could be helpful for documents too, we'll have to catch up with Biometrics later on

Reply:
I have noticed that at a higher resolution (when I bumped up `cameraHeight` > 720px) the multiple frame capture seemed slightly janky on my iPhone 6S. So for better UX might need to just do a single capture. Happy to have a discussion about this with the team, including Biometrics.

Reply:
> I have noticed that at a higher resolution (when I bumped up cameraHeight > 720px) the multiple frame capture seemed slightly janky on my iPhone 6S
I guess it's because we are storing multiple high resolution images.

Reply:
> I have noticed that at a higher resolution (when I bumped up cameraHeight > 720px) the multiple frame capture seemed slightly janky on my iPhone 6S

I guess it's because we are storing multiple high resolution images.

Reply:
Have removed the code for multi capture for now since that is up for discussion with Biometrics. So only a single photo will be captured on capture button press for now.